engine/main.cc

```diff
@@ -31,6 +31,7 @@ void RunServer() {
            << " Port: " << config.apiServerPort << "\n";
 
   // Create logs/ folder and setup log to file
+  std::filesystem::create_directory(config.logFolderPath);
```

Comment:
using `std::filesystem::create_directories` should be better. Seems like it handle create directory recursively.
https://en.cppreference.com/w/cpp/filesystem/create_directory

Also, I'm worry about concatenate path using `/` might not work on windows. Should using `std::filesystem::path`